lang/src/org/partiql/lang/planner/transforms/AstToLogicalVisitorTransform.kt

```diff
@@ -504,6 +526,31 @@ private class FromSourceToBexpr(
         }
 }
 
+/**
+ * This class is used to transform all instances of [PartiqlAst.Expr.CallAgg] into function calls denoted with
+ * COLL_${AGG}, where AGG represents the aggregation call being transformed. This class is designed to be used
+ * AFTER replacing all aggregation functions via [AstToLogicalVisitorTransform.transformAggregations].
+ *
+ * This transformer is designed to be called directly on a [PartiqlAst.Expr.Select] and does NOT recurse into
+ * encountered [PartiqlAst.Expr.Select]'s.
+ */
```

Comment:
I don't believe this is necessary if "COLL_*" are just represented as function calls to begin with. We know they're different when we parse hence the name